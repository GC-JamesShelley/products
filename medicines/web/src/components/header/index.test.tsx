import { shallow } from 'enzyme';
import React from 'react';
import Header from './index';

describe(Header, () => {
  it('should render', () => {
    const component = shallow(<Header title={'MHRA'} />);
    expect(component).toMatchSnapshot();
  });
});
