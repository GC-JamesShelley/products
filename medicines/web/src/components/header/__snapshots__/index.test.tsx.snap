// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header should render 1`] = `
<styled.header>
  <div
    className="wrapper"
  >
    <styled.div>
      <ForwardRef
        href="/"
      >
        <a>
          <SvgMhraLogo />
        </a>
      </ForwardRef>
    </styled.div>
    <h1>
      MHRA
    </h1>
  </div>
</styled.header>
`;
